lib/Horizon.dart

```diff
@@ -0,0 +1,224 @@
+                                                    ),
+                                                ),
+                                            ),
+                                            Image.network(
+                                                dailyForecast.imageId,
+                                                fit: BoxFit.cover,
+                                            ),
+                                            Center(
+                                                child: Text(
+                                                    dailyForecast.getDate(currentDate.day).toString(),
+                                                    style: textTheme.headline4,
+                                                ),
+                                            ),
+                                        ],
+                                    ),
+                                ),
+                                Expanded(
+                                    child: Padding(
+                                        padding: EdgeInsets.all(6.0),
+                                        child: Column(
+                                            crossAxisAlignment: CrossAxisAlignment.start,
+                                            children: <Widget>[
+                                                Text(
+                                                    dailyForecast.getWeekday(currentDate.weekday),
+                                                    style: textTheme.headline5,
+                                                ),
+                                                SizedBox(height: 10.0),
+                                                Text(dailyForecast.description),
+                                            ],
+                                        ),
+                                    )
+                                ),
+                                Padding(
+                                    padding: EdgeInsets.all(6.0),
+                                    child: Text(
+                                        "${dailyForecast.highTemp} | ${dailyForecast.lowTemp} F",
+                                        style: textTheme.subtitle2,
+                                    ),
+                                ),
+                            ],        
+                        ),
+                    );
+                },
+                childCount: 7,
+            ),
+        ); 
+    }
+}
+
+const String baseAssetURL =
+    'https://dartpad-workshops-io2021.web.app/getting_started_with_slivers/';
+const String headerImage = '${baseAssetURL}assets/header.jpeg';
+
+const Map<int, DailyForecast> _kDummyData = {
+  0: DailyForecast(
+    id: 0,
+    imageId: '${baseAssetURL}assets/day_0.jpeg',
+    highTemp: 73,
+    lowTemp: 52,
+    description:
+        'Partly cloudy in the morning, with sun appearing in the afternoon.',
+  ),
+  1: DailyForecast(
```

Comment:
You have a Map with integers as keys. An array implicitly has integers as keys (indexes). Therefore you could probably simplify to an Array data structure vs. a Map.